client/src/main.ts

```diff
@@ -17,6 +18,16 @@ Vue.config.devtools = true
 async function startVue () {
   Vue.prototype.$PAYMENT_PORTAL_URL = await getConfig()
 
+  // configure Keycloak Service
+  await KeyCloakService.setKeycloakConfigUrl(sessionStorage.getItem('KEYCLOAK_CONFIG_PATH'))
+  if (!window.location.pathname.includes('/signin')) {
+    await KeyCloakService.initializeToken(null).then(() => {}).catch(err => {
+      if (err?.message !== 'NOT_AUTHENTICATED') {
+        throw err
```

Comment:
So there's no page reload mechanism here if this fails? (Or user error message.)

Reply:
The scope of the ticket grew a lot here, so some of the error handling isn't entirely fleshed. Some design work was required here. That being said, Scott did mention we should/could use whatever was in place in our other ui's. 
Implementing the expanded error handling / ie dialogs, was going to come under another ticket.